### PR TITLE
Email sending styling

### PR DESCRIPTION
**Pull request by samhstn**
See #187 

Will need to rebase after #231 is merged

### CODE REVIEW

**Review by Danwhy — APPROVED** [review]
:+1: Looks good

File: web/static/assets/fonts/tick-fonts/fontello.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (nelsonic) [inline web/static/assets/fonts/tick-fonts/fontello.svg]: @shouston3 Is this font/tick not available via CDN?

File: web/static/css/app.css [hunk 2]
```diff
@@ -389,3 +393,21 @@ code {
   margin-bottom: 0.3rem;
 }
 
+@font-face {
+  font-family: 'fontello';
+  src: url('/fonts/tick-fonts/fontello.eot');
```
Comment (nelsonic) [inline web/static/css/app.css]: this implies we are loading the _entire_ font library just for the `tick` is that correct?

**Review by nelsonic — COMMENTED** [review]
@shouston3 looks good. left a couple of questions. please feedback then we can merge. thanks.

**Comment by shouston3** [comment]
@nelsonic will look at changing this

**Review by nelsonic — APPROVED** [review]
@shouston3 looks good. thanks. merging. 🎉

**Pull request merged**